core/systems/threading/shared_thread.gd

```diff
@@ -145,6 +169,17 @@ func remove_process(method: Callable) -> void:
 
 
 func _run() -> void:
+	# Update the thread ID
+	mutex.lock()
+	tid = LinuxThread.get_tid()
+	mutex.unlock()
+	logger.info("Started thread with thread ID: " + str(LinuxThread.get_tid()))
+	
+	# Set the thread priority
```

Comment:
Comment this better, saying check to see if niceness is set, then set the priority or something